beta-src/src/state/interfaces/GameOverviewResponse.ts

```diff
@@ -8,6 +8,7 @@ export type Member = {
 
 interface GameOverviewResponse {
   anon: string;
+  date: string;
```

Comment:
are you sure this comes back from the api response?

Reply:
Yea, the api.php was also changed so that the game overview response object returns the necessary info. Instead of date now, its season and year!